src/Group.php

```diff
@@ -38,13 +38,20 @@
  **/
 class Group extends CommonTreeDropdown
 {
+    use Glpi\Features\Clonable;
+
     public $dohistory       = true;
 
     public static $rightname       = 'group';
 
     protected $usenotepad  = true;
 
 
+    public function getCloneRelations(): array
```

Comment:
I think it could make sense to clone the `Group_User` relations. See what other reviewers think about it.

Reply:
It has not been cloned in 9.5.
In my opinion, the point of groups is that each one brings together different users?

Reply:
Indeed.